Fix compatibility with cmake>=4.
Do not install patch .orig files

--- CMakeLists.txt~	2013-06-19 01:16:00.000000000 +0200
+++ CMakeLists.txt	2013-06-20 17:39:16.057129720 +0200
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 
 set(catkin_EXTRAS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
@@ -18,6 +18,7 @@
 install(DIRECTORY cmake
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
   USE_SOURCE_PERMISSIONS
+  PATTERN "*.orig" EXCLUDE
   PATTERN "catkinConfig.cmake.in" EXCLUDE)
 
 catkin_python_setup()
